roofit/roofit/src/RooFitComputeLib.cxx

```diff
@@ -0,0 +1,60 @@
+/* RooFitComputeLib.cxx - created September 2020
+ */
+#include "RooFitComputeLib.h"
+#include "RooVDTHeaders.h"
+#include "BatchHelpers.h"
+#include "RunContext.h"
+
+using namespace BatchHelpers;
+
+namespace RooFitCompute {
+
+  /// Small helping function that determines the sizes of the batches and decides
+  /// wether to run the High performing or the Generic computation function.
```

Comment:
This belongs to the function, not to the struct.

Reply:
Fixed